Src/library/memory_analyzer/widgets/ma_window_imp.e

```diff
@@ -1041,6 +1055,11 @@ feature {NONE} -- Implementation
 		deferred
 		end
 
+	dpi_resize_history (a_dpi, a_x, a_y, a_width, a_height: INTEGER)
+			-- Called by `dpi_changed_actions' of `eiffel_history'.
+		deferred
```

Comment:
maybe , you should have an effective feature calling `resize_history `

Reply:
Done.